presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator resource group aggregation issue related to WAITING_FOR_PREREQUISITE queries.
+* Fix Iceberg $files table in case of column dropping.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory overcounting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an error in ``tdigest_agg``.
+* Add `to_base32` and `from_base32` varbinary functions.
+* Add support for TRUNCATE TABLE statement.
```

Comment:
Truncate table should be in double back-tick quotes like

```
``TRUNCATE TABLE``
```

Should also link to relevant documentation here.

Reply:
added quotes but I can't find the document for "truncate" in https://prestodb.io/docs/current/
is that something that has to be added ? @rschlussel

Reply:
I checked the commit and it does include a doc change adding TRUNCATE. But we can't add a link here before the docs are published ?